### PR TITLE
Add --unit argument to pay() through CLI

### PR DESCRIPTION
**Pull request by rklf**
- Add `--unit` (time unit) argument to pay through CLI
- Replace deprecated `utcnow` method
- Add `venv/` to `.gitignore`
- Update `README.md` to reflect the changes

### CODE REVIEW

File: paybybot3/__main__.py [hunk 1]
```diff
@@ -131,9 +130,10 @@ def _alert(config, location):
 @click.option("--location", required=True, type=str)
 @click.option("--rate", required=True, type=int)
 @click.option("--duration", required=True, type=str)
+@click.option("--unit", default="Days", type=click.Choice(["Days", "Minutes"]), help="Default: Days")
```
Comment (rklf) [inline paybybot3/__main__.py]: Please add `Hours` as possible units. @HamletDuFromage

**Review by rklf — COMMENTED** [review]
Thanks for this

**Comment by rklf** [comment]
Thanks.

**Comment by dev590t** [comment]
@HamletDuFromage 
In response to https://github.com/rklf/paybybot3/issues/8#issuecomment-2169853270, I have tried in a real test, the code run perfectly.

**Pull request merged**